main.go

```diff
@@ -1,9 +1,9 @@
 package main
 
 import (
-	"github.com/Alxus228/Brainfuck/brainfuck"
+	"github.com/Alxus228/Brainfuck-Interpretator/brainfuck"
 )
 
 func main() {
-	brainfuck.Interpetate("++++++++[>++++[>++>+++>+++>+<<<<-]>+>+>->>+[<]<-]>>.>---.+++++++..+++.>>.<-.<.+++.------.--------.>>+.>++.")
+	brainfuck.Brainfuck("++++++++[>++++[>++>+++>+++>+<<<<-]>+>+>->>+[<]<-]>>.>---.+++++++..+++.>>.<-.<.+++.------.--------.>>+.c>p.")
```

Comment:
`brainfuck` should DO something. Pls use verb instead of `Brainfuck`

Reply:
Done